### DISCUSSION TITLE
Tasks plugin: Utilizing Teams

### DISCUSSION THREAD

**Discussion by empx92** [kimai/Paid-plugins#135] [General]
When creating a task and assigning it to a team, such as the 'Administration' team encompassing all office staff (User 1, User 2, User 3), all members can view the task on the dashboard. However, an unusual behavior arises when assigning the task to an additional team member (e.g., User 4). In this case, the task disappears from the dashboard for all team members.

While this behavior has its practicality, it would be a valuable feature to have the ability to display the standard 'Tasks' page as a widget on the main dashboard. This would provide users with three distinct views: 'My Tasks,' 'Scheduled Tasks' (tasks planned but not yet assigned), and 'Scheduled & Assigned Tasks.'

An additional, advanced feature could visually represent when a user is assigned to tasks spanning multiple days or weeks. This would allow for a quick overview of team members' availability, enhancing overall task management efficiency."

This is neither a question nor a complaint on my part: instead, I simply want to share my thoughts with the community.

**Comment by kevinpapst** [comment]
> This would allow for a quick overview of team members' availability, enhancing overall task management efficiency.

Do you know there are two reports specific for tasks and their estimations?

> In this case, the task disappears from the dashboard for all team members.

Why would you want that task to still appear on the Dashboard of the team members, which do not work on it?

Not sure without checking the code, but there is one permissions that allows to start tasks from "pending tasks" widget, which might help solving this specific issue. But I am not sure if it influences visibility. This was added, because certain tasks should not be assigned to specific users, but everyone should be able to start and stop time for them.

> visually represent when a user is assigned to tasks spanning multiple days or weeks

How? Any idea? I am happy about good ideas that make it better for everyone.

**Reply by empx92** [reply]
> Do you know there are two reports specific for tasks and their estimations?

Yeah i know the views "Task per user" and "Team Tasks"

> Why would you want that task to still appear on the Dashboard of the team members, which do not work on it?

In my case the Users are outside the company and they travel in a radius about 60km, so it is not always easy to know where the persons are and what they "do". So if there are "urgent" tasks may our secretary team will call after the employee if he has not forgotten to fulfill the task ;) Also currently we are in transformation in the use of the tracking software, which in my opinion is a process of time in every company - so with many employees there is some need of control. As mentioned the standard view of the "Task" link on the mainmenu does everything needed but a widget as mantioned could even enhance it more.

> How? Any idea? I am happy about good ideas that make it better for everyone.

I could do some "mockup" work with the idea, if it would be useful for you.
I am thinking about a view of the calender like in the time sheets (month, week, eventually even the day. Inside this view the Projekt could be displayed in a time line with the assigned team or users. the second view could be a list with "available" employees. In our case we try to manage as many as possible within kimai. but for example to plan the projects which are already all available in kimai we need to use paper and pen. the whole management of our employees is running trough paper & pen es we need to have an overview per week. we also mark "employees" which are on a project with others but could be used for emergency and so on. I think this management is one of the hardest things if you have some employees.
Most things can get solved when you "break" out through an "old" workflow, but some things are not solved yet ;) I would be happy if i am into Twig/Symfony programming but the point is that i can use HTML, PHP and CSS, so it is hard to setup my ideas like in a custom plugin. There isn't enough time to get into that specific workflow which is needed therefore. 

Offtopic: I wanted to ask if you did get my inquiry per Mail from 19th February?

**Reply by kevinpapst** [reply]
> Offtopic: I wanted to ask if you did get my inquiry per Mail from 19th February?

Not that I know

**Reply by empx92** [reply]
Was initial sent at support@kimai.com like i read on the website and now sent again to info@keleo.de is that right?

**Reply by kevinpapst** [reply]
Please never sent it to kimai.com - kimai.org is correct
